cmd/tailscale/cli/cli.go

```diff
@@ -207,7 +207,12 @@ change in the future.
 			debugCmd,
 			driveCmd,
 			idTokenCmd,
-		},
+		}, (func() []*ffcli.Command {
+			if !envknob.UseWIPCode() {
+				return nil
+			}
+			return []*ffcli.Command{advertiseCmd}
+		}())...),
```

Comment:
move the conditional func into `advertise.go` like `funnelCmd()`, `serveCmd()`, and `exitNodeCmd()`

Reply:
unfortunately the `append()` still needs to be there because this func might return nil -- those others always return a non-nil `Command`, the func just determines what goes in it. I'll move the inline func though